MASFoundation/Classes/_private_/services/network/MASNetworkingService.m

```diff
@@ -1444,11 +1445,17 @@ - (void)putTo:(NSString *)endPoint
     //
     // Default types
     //
+    NSInteger responseType;
+    if ([endPoint  isEqual: @"/connect/device/renew"]) {
+        responseType = MASRequestResponseTypeTextPlain;
+    } else {
+        responseType = MASRequestResponseTypeJson;
+    }
```

Comment:
It would be better to not make a response type change here.
Better idea would be add proper response type parameters to the certificate renew http call in the first place.